src/client/time/browser_clock.ts

```diff
@@ -0,0 +1,29 @@
+import { CancelTimer } from '../../shared/time/clock'
+import { Clock } from '../../shared/time/clock'
+
+function setTimeout(cb: (...args: any[]) => void, seconds: number): CancelTimer {
+  const handle = window.setTimeout(cb, seconds * 1e3)
+  return window.clearTimeout.bind(null, handle)
```

Comment:
Can you change this to undefined? That is the preferred in typescript and is consistent with the rest of the codebase. The same for the one's below.